docs/aso/metadata-v1.1.1.md

```
@@ -182,22 +192,61 @@ v1.1.0 がメタデータ起因でリジェクトされた場合は、本パッ
 かつ「中断してもその場から再開できる」は**実装済みの差別化点**。§3-0 の原則どおり、
 実測が最も良い語は不確実な2枠目ではなく**確実に入る1枠目**に置く。
 
+### 3-5. #281 で追加した2語・外した1語（2026-08-25 の実測が根拠）
+
+`麻雀`（#106 / PR #259）・`数独`（#262 / PR #268）は 2026-08-24 に **`release/v1.1.1` へ直接マージ**され、
+**次に出す版に入っている**のに、日本語ロケールのタイトル・サブタイトル・キーワードの**どこにも無かった**。
+#178 で大富豪・麻雀ソリティアについて直したのと**同一の欠陥の3回目**（8→10→12）。
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)